data/yam/autoyast/support_images/autoyast_sles15sp4_sp6_media_install_textmode_default_patterns_aarch64.xml

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0"?>
```

Comment:
Please don't follow this name style except these three product profiles are completely same. Suggest to name it as autoyast_sles15sp6_media_install_textmode_default_patterns_aarch64.xml  So for other files.

Reply:
Solved here c654766b3